packages/account/src/Components/additional-kyc-info-modal/__test__/additional-kyc-info-form.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { render, fireEvent, screen, waitFor } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import { AdditionalKycInfoForm } from '../additional-kyc-info-form';
+
+// Mock useSettings hook to prevent errors
+jest.mock('../../../../../api/src/hooks', () => ({
+    useSettings: () => ({
+        update: jest.fn(),
+        mutation: { isLoading: false, isSuccess: false, error: null, isError: false },
+    }),
+}));
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    generateValidationFunction: jest.fn(),
+}));
+
+const mock_store = mockStore({});
+
+describe('AdditionalKycInfoForm', () => {
+    const setError = jest.fn();
+    it('should render the form fields', () => {
+        render(
+            <StoreProvider store={mock_store}>
+                <AdditionalKycInfoForm setError={setError} />
+            </StoreProvider>
+        );
+
+        // Assert that form fields are rendered
+        expect(screen.getByTestId('place_of_birth')).toBeInTheDocument();
```

Comment:
lets follow the naming rule for testIDs - dt_place_of_birth

Reply:
Thanks @amina-deriv 
Will update